10.0/s2i/bin/run

```diff
@@ -1,2 +1,12 @@
 #!/bin/bash
-exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0
+
+run_wildfly() {
+  if [ "$DEV_MODE" == true ]; then
+    exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0 -Dhot.deploy.enabled=true -DENABLE_JPDA=true
+  else
+    exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0
+  fi
+}
+
+run_wildfly
```

Comment:
Ok noted, I will remove it. Do we have any doc with best practices about how to make a flexible "run" and "assemble" file